python/GafferImageUI/CatalogueSelectUI.py

```diff
@@ -51,11 +51,17 @@ def __imageNames( plug ) :
 
 def __imagePresetNames( plug ) :
 
-	return IECore.StringVectorData( [ "Selected" ] + __imageNames( plug ) )
+	return IECore.StringVectorData(
+		[ "Selected", "Output 1", "Output 2", "Output 3", "Output 4" ] +
+		[ "Image Name/" + i for i in __imageNames( plug ) ]
```

Comment:
This feels a bit better to me if the outputs are in a submenu too, and we drop the "Name" part from the "Image Name" submenu.

![image](https://user-images.githubusercontent.com/1133871/192517208-62bfe988-a8db-408a-ad5b-73cd63847534.png)